proxy/src/proxy.rs

```diff
@@ -280,7 +280,7 @@ pub async fn handle_client<S: AsyncRead + AsyncWrite + Unpin>(
 
     // check rate limit
     if let Some(ep) = user_info.get_endpoint() {
-        if !endpoint_rate_limiter.check(ep) {
+        if !endpoint_rate_limiter.check(ep, 1) {
```

Comment:
It looks like you do this check twice: here and in `auth_quirks`.

Reply:
This is the endpoint rate limiter, whereas in auth_quirks it's the auth rate limiter

Reply:
Hm, and why we don't have it for serverless?

Reply:
Oversight 🤷

Reply:
Could you please add rate limiting to sql-over-http flow as well?